apps/editors/views.py

```diff
@@ -558,8 +558,8 @@ def review(request, addon):
     form = forms.get_review_form(request.POST or None, request=request,
                                  addon=addon, version=version)
 
-    queue_type = (form.helper.review_type if form.helper.review_type
-                  != 'preliminary' else 'prelim')
+    queue_type = (form.helper.review_type
+                  if form.helper.review_type != 'preliminary' else 'prelim')
```

Comment:
I think this would make more sense with the cases reversed, the formatting is definitely trickier with the line lengths though.

```py
    queue_type = ('prelim' if form.helper.review_type == 'preliminary'
                  else form.helper.review_type)